docs/bindings.md

```diff
@@ -55,6 +55,8 @@ in the [can-view-scope scope]:
 <my-component {$child-attr}="value"/>
 ```
 
+> __Note:__ If the value is an object, changes to its properties will still be updated. See <a href="#OneWayBindingWithObjects">One Way Binding With Objects</a>.
```

Comment:
If value being passed to the component is an object, changes to the objects properties will still be visible to the component.  Objects are passed by reference.   See <a href="#OneWayBindingWithObjects">One Way Binding With Objects</a>.